lib/openstack_api/openstack_flavor.py

```diff
@@ -0,0 +1,132 @@
+from typing import List, Dict, Union, Optional
+from openstack.compute.v2.flavor import Flavor
+from openstack_api.openstack_wrapper_base import OpenstackWrapperBase
+
+
+class OpenstackFlavor(OpenstackWrapperBase):
+    def list_flavor(self, cloud_account: str) -> List[Flavor]:
+        """
+        Get list of flavors from either production or development cloud
+        :param cloud_account: Name of cloud to use
+        :return: List of Flavor Objects
+        """
+        with self._connection_cls(cloud_account) as conn:
+            return conn.list_flavors()
+
+    def get_flavor(self, cloud_account: str, flavor_name: str) -> Flavor:
+        """
+        Get information for a specific flavor
+        :param cloud_account: Name of cloud to use
+        :param flavor_name: name of the flavor to get
+        :return: Flavor object or None if unsuccessful
+        """
+        with self._connection_cls(cloud_account) as conn:
+            return conn.get_flavor(flavor_name)
+
+    def get_missing_flavors(self, source_cloud: str, dest_cloud: str) -> List[str]:
+        """
+        Get list of flavors missing in Dev cloud based on the flavors currently in
+        production
+        :param source_cloud: Cloud account for source cloud
+        :param dest_cloud: Cloud account for destination cloud
+        :return: List of names of missing flavors or empty list
+        """
+        # list flavors from prod
+        source_flavors = self.list_flavor(source_cloud)
+
+        # list flavors from dev
+        dest_flavors = self.list_flavor(dest_cloud)
+
+        # Prepare list of flavors that are in prod and dev
+        source_flavor_names = []
```

Comment:
We can remove the two for loops below and use list comprehension instead 
```
source_flavor_names = [source.name for source in source_flavors]
dest_flavor_names = [dest.name for dest in dest_flavors]
```

Also we can go straight to a set rather than doing list -> set
```
source_flavor_names = {source.name for source in source_flavors}
dest_flavor_names = {dest.name for dest in dest_flavors}
```

This allows us to get rid of the `set(source_flavor_names)` and `set(dest_flavor_names)` casts below